src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2786,4 +2786,10 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Remove_async_modifier" xml:space="preserve">
     <value>Remove 'async' modifier</value>
   </data>
+  <data name="Pull_0_up_to_new_base_class" xml:space="preserve">
+    <value>Pull '{0}' up to new base class</value>
```

Comment:
if you move to support selecting multiple memberes, you can just say `Pull member(s) up to new base class`.

Reply:
And when on a class declaration, it would be teh same, except `Pull members up to new base class...` to indicate a dialog is coming.

Reply:
I changed all of it to `Pull member(s) up to new base class...` for now, since it will always invoke a dialog. The only different will be what is preselected in the dialog for the user.